lib/scrape.js

```diff
@@ -165,6 +166,45 @@ async function findOrganization({
   return null
 }
 
+async function findWiki(name) {
+  const APIurl = wdk.searchEntities({
+    search: name,
+    format: 'json',
+  })
+  const srch = await fetch(APIurl)
+  const response = await srch.json()
+  if (response && response.search.length) {
+    const wikidataurl = response.search[0].url
+    const wikidataid = response.search[0].id
+    const entityAPIurl = wdk.getEntities({
+      ids: [wikidataid],
+      props: ['sitelinks'],
+      format: 'json',
+    })
+
+    const sitelinks = await fetch(entityAPIurl)
+    let sitelink = await sitelinks.json()
+    sitelink = sitelink['entities'][wikidataid]['sitelinks']
```

Comment:
In general I think doing `sitelink.entities[wikidataid].sitelinks` is more appropriate (and I think that's what we've been doing in other places), but either works. Is there an ESLint rule for this?